spec/lib/tasks_helpers/metrics_spec.rb

```diff
@@ -0,0 +1,102 @@
+describe 'Metrics' do
+  let(:header) do
+    %w[
+      tenant
+      portfolio_count
+      product_count
+      portfolio_share_count
+    ].collect(&:titleize)
+  end
+
+  let(:tenant) do
+    FactoryBot.create(
+      :tenant,
+      :name            => 'tenant_portfolio_shared_context',
+      :external_tenant => 'external_tenant_portfolio_shared'
+    )
+  end
+
+  def create_portfolio
+    portfolio = FactoryBot.create(:portfolio)
+    FactoryBot.create(:access_control_entry, :aceable => portfolio)
```

Comment:
Same with these, they can just be `create(:portfolio)` and `create(:access_control_entry, :aceable => portfolio)`.